### PR TITLE
11.0 mig website_sale_product_brand

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa

### CODE REVIEW

File: website_sale_product_brand/README.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+User can assign a nice logo with brand description.
+
+- After configuring the brand, user can assign a particular brand to a particular products.
+
+Based on this configuration, you will see the menuitem shop by brand next to shop menu.
+It will show all the brands and once you select that brand it will show product's which
+is related to this brand.
+
+The blog here explains the HOWTO :
+    http://www.serpentcs.com/serpentcs-odoo-ecommerce-shop-brands-contribution
```
Comment (chienandalu) [inline website_sale_product_brand/README.rst]: Do this link give any relevant info? The README.rst info should be the more self contained possible

File: website_sale_product_brand/README.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+==========================
+
+Description
+===========
+
+This module was written to extend the functionality of product filtering on website.
+It will allow you to filter product based on its brand.
+
+While shopping online, we have seen various eShops having a feature to shop by brands
+which ODOO does not yet provide officially.Website Sale Product Brand fills the gap at certain
```
Comment (chienandalu) [inline website_sale_product_brand/README.rst]: Need a space between *officially.* and *Website*

File: website_sale_product_brand/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# © 2016 Serpent Consulting Services Pvt. Ltd. (http://www.serpentcs.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Brand Filtering in Website',
+    'category': 'e-commerce',
+    'author': "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.serpentcs.com',
```
Comment (chienandalu) [inline website_sale_product_brand/__manifest__.py]: https://github.com/OCA/e-commerce

File: website_sale_product_brand/README.rst [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+extent and by providing basic search by brands, thus reducing end-user’s efforts in 
+searching the products he/she wants to purchase.
+
+Installation
+============
+
+To install this module, you need to install following module:
+->https://github.com/OCA/product-attribute/tree/11.0/product_brand
```
Comment (chienandalu) [inline website_sale_product_brand/README.rst]: Dependencies are already documented in the manifest file

**Review by chienandalu — CHANGES_REQUESTED** [review]
You need to declare oca_dependencies.txt with product-attribute repo.
You should also use the new OCA readme system

File: website_sale_product_brand/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply website_sale_product_brand/README.rst]: Indeed no links outside OCA

File: website_sale_product_brand/README.rst (hunk 4 above)
Comment (pedrobaeza) [reply website_sale_product_brand/README.rst]: That's not true. You have to indicate dependencies outside of the current repo, which is the case.

**Comment by pedrobaeza** [comment]
@ernestotejeda you have to include in oca_dependencies.txt file in the root the repository `product-attribute` for having correct Travis and runbot.

File: website_sale_product_brand/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# © 2016 Serpent Consulting Services Pvt. Ltd. (http://www.serpentcs.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Brand Filtering in Website',
+    'category': 'e-commerce',
+    'author': "Serpent Consulting Services Pvt. Ltd.,"
+              "Odoo Community Association (OCA)",
```
Comment (rafaelbn) [inline website_sale_product_brand/__manifest__.py]: @JayVora-SerpentCS @pedrobaeza we are maintaining this module from v9 with fix + improvement + migrating.  Do you think is fair to add here Tecnativa o not?

**Review by rafaelbn — COMMENTED** [review]
Set as WIP cc @Tecnativa

**Comment by rafaelbn** [comment]
cc @Tecnativa

File: website_sale_product_brand/__manifest__.py (hunk 5 above)
Comment (JayVora-SerpentCS) [reply website_sale_product_brand/__manifest__.py]: Fine to add @rafaelbn ! Thanks.

File: website_sale_product_brand/README.rst [hunk 6]
```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (MeetKD) [inline website_sale_product_brand/README.rst]: Png

File: website_sale_product_brand/views/product_brand.xml [hunk 7]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```
Comment (MeetKD) [inline website_sale_product_brand/views/product_brand.xml]: <odoo>

**Review by MeetKD — CHANGES_REQUESTED** [review]

File: website_sale_product_brand/README.rst (hunk 6 above)
Comment (pedrobaeza) [reply website_sale_product_brand/README.rst]: Better to use new README by fragments system. @ernestotejeda check other of your PRs where I have done it.

**Comment by rafaelbn** [comment]
Please @ernestotejeda could you review this PR , we can't review it. Thanks! :smile:

**Review by MeetKD — APPROVED** [review]
LGTM :+1:

File: website_sale_product_brand/__manifest__.py [hunk 8]
```diff
@@ -19,6 +19,10 @@
         "security/ir.model.access.csv",
         "views/product_brand.xml",
     ],
+    'demo': [
+        "demo/assets.xml",
+        "data/brand_data.xml"
```
Comment (yajo) [inline website_sale_product_brand/__manifest__.py]: Demo data goes into demo folder. Also please append trailing `,`

File: website_sale_product_brand/static/src/js/tour.js [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+    var core = require("web.core");
+    var tour = require("web_tour.tour");
+
+    var _t = core._t;
+
+    var steps = [
+        {
+            trigger: "a[href='/page/product_brands']",
+            content: _t("Go to 'Product brand' page"),
```
Comment (yajo) [inline website_sale_product_brand/static/src/js/tour.js]: If this tour is only for testing (not for guiding first-time users), then don't translate strings.

File: website_sale_product_brand/tests/__init__.py [hunk 10]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```
Comment (yajo) [inline website_sale_product_brand/tests/__init__.py]: :fire:

File: website_sale_product_brand/tests/test_ui.py [hunk 11]
```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import HttpCase
+
+
+class UICase(HttpCase):
+    def setUp(self):
+        """Ensure website lang is en_US."""
+        super(UICase, self).setUp()
```
Comment (yajo) [inline website_sale_product_brand/tests/test_ui.py]: :fire: This method does nothing? :thinking:

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by JGarcia-Panach** [comment]
Functional review 👍

**Review by JGarcia-Panach — APPROVED** [review]
Fucntional review 👍

**Review by rafaelbn — APPROVED** [review]
Please @ernestotejeda review technical commments, Functionally is OK :+1:

File: website_sale_product_brand/demo/product_brand_demo.xml [hunk 12]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product_brand_apple_demo" model="product.brand">
+        <field name="name">Apple</field>
+    </record>
+</odoo>
```
Comment (chienandalu) [inline website_sale_product_brand/demo/product_brand_demo.xml]: Add an EOF empty line

File: website_sale_product_brand/demo/product_product_demo.xml [hunk 13]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product.product_product_7" model="product.product">
+        <field name="product_brand_id" ref="product_brand_apple_demo"/>
+    </record>
+    <record id="product.product_product_9" model="product.product">
+        <field name="product_brand_id" ref="product_brand_apple_demo"/>
+    </record>
+</odoo>
```
Comment (chienandalu) [inline website_sale_product_brand/demo/product_product_demo.xml]: Add an EOF empty line

**Comment by pedrobaeza** [comment]
As @Yajo's comments have been honored, I merge this.

**Pull request merged**